docs/sources/installation/mac.md

```diff
@@ -9,49 +9,50 @@ weight = 500
 +++
 
 
-# Installing on macOS
+# Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
+This page provides instructions to help you install Grafana on macOS. You can either install using Homebrew or a binary tar file.
```

Comment:
Is the binary tar file still an option? I do not see it on the [Download Grafana](https://grafana.com/grafana/download?platform=mac) page.

Reply:
@DanCech ?

Reply:
*pokes* @DanCech